bigtable-hbase-replication/src/main/java/com/google/cloud/bigtable/hbase/replication/adapters/HbaseBigtableWALEntryAdaptor.java

```diff
@@ -0,0 +1,8 @@
+package com.google.cloud.bigtable.hbase.replication.adapters;
+
+/*
+HbaseBigtableWALEntryAdaptor provides BigtableWALEntry adaptor for any WAL entry
+ */
+public interface HbaseBigtableWALEntryAdaptor {
```

Comment:
This adaptor interface is not really "adapting". To define the adaptor correctly ( newType adapt(OldType old)) you need to depend on the "WAL.Entry" which defeats the purpose of creating BigtableWALEntry. 

This package should use BigtableWALEntry everywhere, it is the callers responsibility to figure out how to create it. We can delete this file here.

Reply:
Deleted the file

Reply:
upcoming PR.